tests/alchemyapp/models.py

```diff
@@ -17,7 +17,15 @@ class StandardModel(Base):
     __tablename__ = 'StandardModelTable'
 
     id = Column(Integer(), primary_key=True)
-    foo = Column(Unicode(20))
+    foo = Column(String(20))
```

Comment:
Why change the data type? Given that we’re passing unicode data, instructing SQLAlchemy that the column type is Unicode should be correct?

Tests pass with both String and Unicode.

In any case, I don’t think this should be grouped with the change “Add SQLAlchemy get_or_create support”. If there’s a reason to use String instead of Unicode, can you explain it? Perhaps even make a PR to change all instances at once?

Reply:
I left a comment in this PR a few months ago, regarding some warnings when saving strings into unicode fields on python 2.7 environment.

Seems its safe to keep the unicode fields now.